docs/CppInteroperability/CppInteroperabilityStatus.md

```diff
@@ -0,0 +1,102 @@
+| Top-level functions                         | Yes    |
+| Enumerations                                | Yes. That includes `enum class`  |
+| Struct / Class types                        | Yes - as value types, except for types without a copy constructor. Partial experimental support for importing a C++ struct/class as a reference type |
+| Typedefs / Type aliases                     | Yes    |
+| Global Variables                            | Yes    |
+| Namespaces                                  | Yes    |
+| Inline Namespaces                           | Yes, with some known issues |
+| Exceptions                                  | No  |
+| Fields                                      | Yes |
+| Member functions                            | Yes. Some value category overloads aren't imported |
+| Virtual Member Functions                    | No |
+| Operators                                   | Yes, with some known issues    |
+| Subscript Operators                         | Yes |
+| Constructors                                | Yes. That includes implicit constructors    |
+| Destructor                                  | Yes. C++ destructors are invoked automatically when the value is no longer used in Swift |
+| Copy constructor / copy assignment operator | Yes. Swift invokes the underlying copy constructor when copying a C++ value |
+| Move constructor / move assignment operator | No    |
+| Base class member functions / operators     | Yes, with some known issues |
+| Function templates                          | Yes    |
+| Class templates                             | Yes    |
+| Dependent types                             | Partially: imported as Any |
+| Availability Attributes                     | Yes    |
+
+
+The following C++ code patterns or language features have specific mappings to Swift language features when imported in Swift:
+
+
+| **C++ Language Feature**                             | **Imported Into Swift**                                                                  |
+|------------------------------------------------------|------------------------------------------------------------------------------------------|
+| `get`/`set` member functions                         | Imported as computed property (since Swift-5.7)                                          |
+| `const`/non-`const` member function overload set     | Both overloads are imported as a method, with non-`const` method being renamed to `mutating…`. (since Swift-5.7). This will change in a future version of Swift                        |
+
+
+Unless stated otherwise (i.e., imported reference types) all Swift features work with imported types. For example: use in generic contexts, protocol conformance, extensions, etc.
+
+
+### C++ Standard Library Support
+
+Libc++ can be imported and used from Swift.
+
+This status table describes which of the following C++ standard library features have some experimental support for using them in Swift:
+
```

Comment:
We can also add "This is not a comprehensive list and libc++ APIs that use the above supported C++ features could be imported into Swift." (If this is the case.)